data/json/items/tools.json

```diff
@@ -5808,15 +5812,15 @@
     "description": "This is a sealed glass jar containing tomatoes.  Use to open and eat to enjoy.",
     "price": 650,
     "material": ["glass", "veggy"],
-    "weight": 600,
+    "weight": 1050,
     "volume": 2,
     "bashing": 4,
     "category" : "food",
     "use_action": {
         "type": "transform",
         "msg": "You open the jar, exposing it to the atmosphere.",
         "target": "can_tomato",
-        "target_charges": 3,
+        "target_charges": 2,
```

Comment:
I just noticed that the recipe for canned tomato actually takes just one. It should probably take two (because a single tomato should take the whole jar).